### ISSUE TITLE
Upgrade containerd to include a fix for CRI issue #1280

### ISSUE THREAD

**Issue by matlec** [rancher/k3s#931]
**Version:**
k3s version v0.10.0-rc1 (ccb1d81f)

**Describe the bug**
Since we upgraded from k3s 0.8.1, several pods (including the pods deployed by the official PostgreSQL helm chart) fail to run with the following error message:

```
Error: failed to create containerd task: OCI runtime create failed: container_linux.go:346: starting container process caused "exec: \"sh\": executable file not found in $PATH": unknown`
```

**To Reproduce**
- use Helm to install PostgreSQL from stable/postgresql

**Additional context**
containerd [Issue #3597](https://github.com/containerd/containerd/issues/3597) describes the original bug in containerd which got fixed by this [pull request](https://github.com/containerd/containerd/pull/3599) and [cherry-picked](https://github.com/containerd/cri/pull/1282) to the 1.3 release branch.

Please bump containerd/cri to a revision that includes this fix. Thank you!

**Comment by mshade** [comment]
I can verify this bug is also present in k3s v0.9.1.

**Comment by davidnuzik** [comment]
I was able to also reproduce on v0.10.0-rc2:
`error: Internal error occurred: error executing command in container: failed to exec in container: failed to start exec "5e2d58d5e58552a9f173d8ac1aa31769529c09ca25d6a03611d0f3455f2173a3": OCI runtime exec failed: exec failed: container_linux.go:346: starting container process caused "exec: \"echo ok\": executable file not found in $PATH": unknown`

**Comment by davidnuzik** [comment]
Appears to be fixed with bump in containerd version in master.

**Comment by davidnuzik** [comment]
Fixed in v0.10.0 release.
